godot/Cargo.toml

```diff
@@ -9,11 +9,11 @@ categories = ["game-engines", "graphics"]
 
```

Comment:
Your seem to accidentally revert several unrelated changes from `master`. Please undo those changes.